boards/nordic/nrf52840dk/src/io.rs

```diff
@@ -24,7 +24,7 @@ enum Writer {
 static mut WRITER: Writer = Writer::WriterUart(false);
 
 fn wait() {
```

Comment:
Wow, that's one magic function! Tried to capture what I think it does in a comment, feel free to adjust.

```suggestion
// Wait a fixed number of cycles to avoid missing characters over the RTT console
fn wait() {
```